bin/mango-submit

```diff
@@ -78,7 +78,7 @@ echo "Using SPARK_SUBMIT=$SPARK_SUBMIT"
 "$SPARK_SUBMIT" \
   --class org.bdgenomics.mango.cli.VizReads \
   --conf spark.serializer=org.apache.spark.serializer.KryoSerializer \
-  --conf spark.kryo.registrator=org.bdgenomics.adam.serialization.ADAMKryoRegistrator \
+  --conf spark.kryo.registrator=org.bdgenomics.mango.serialization.MangoKryoRegistrator \
```

Comment:
Hmm weird, I've already rebased earlier today and just tried it again. It says rebase was successful. Is there a way to remove those commits in the PR?